packages/static-wado-webserver/lib/dicomWebServerConfig.mjs

```diff
@@ -10,11 +12,23 @@ const { staticWadoConfig } = StaticCreator;
 const { dicomWebServerConfig } = ConfigPoint.register({
   dicomWebServerConfig: {
     configBase: staticWadoConfig,
+    dicomWebScpConfig,
     helpShort: "dicomwebserver",
     helpDescription: "Serve up the static wado files and optionally a web client as a web server on the local machine.",
     studyQuery: "studiesQueryByIndex",
     clientDir: "~/ohif",
     port: 5000,
+    externalViewer: {
```

Comment:
This should be called cors, and should be an array of objects, each of which looks like this individual item.